tests/testthat/test-carehomes-support.R

```diff
@@ -303,6 +303,42 @@ test_that("Can compute initial conditions", {
 })
 
 
+test_that("Can control the seeding", {
+  p <- carehomes_parameters(sircovid_date("2020-02-07"), "london",
+                            initial_I = 50)
+  expect_equal(p$initial_I, 50)
+
+  mod <- carehomes$new(p, 0, 10)
+  info <- mod$info()
+
+  initial <- carehomes_initial(info, 10, p)
+  expect_setequal(names(initial), c("state", "step"))
+  expect_equal(initial$step, p$initial_step)
+
+  initial_y <- mod$transform_variables(initial$state)
+
+  expect_equal(initial_y$N_tot3, sum(p$N_tot))
+  expect_equal(initial_y$N_tot2, sum(p$N_tot))
+  expect_equal(initial_y$N_tot, p$N_tot)
+
+  expect_equal(rowSums(initial_y$S) + drop(initial_y$I_A),
+               p$N_tot)
+  expect_equal(drop(initial_y$I_A),
+               append(rep(0, 18), 50, after = 3))
+  expect_equal(initial_y$T_sero_pre[, 1, 1, ],
+               append(rep(0, 18), 50, after = 3))
+  expect_equal(initial_y$T_PCR_pos[, 1, 1, ],
+               append(rep(0, 18), 50, after = 3))
+  expect_equal(initial_y$react_pos, 50)
+
+  ## 42 here, derived from;
```

Comment:
Just a comment but I'm not sure how to get the name of the states !=0 to update the comment (as now we have 64 non zero states initially, not 42